projects/scrypt/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Build the fuzzers and project source code
+cargo fuzz build
+
+# Copy built fuzzer binaries to $OUT
+find $SRC/password-hashes/fuzz/target/x86_64-unknown-linux-gnu/release -maxdepth 1 \
+    -type f -perm -u=x -exec cp {} $OUT \;
```

Comment:
What will happen, if in the future we add fuzz targets for other crates? If it runs all generated binaries, then I think it's better to rename the folder from `projects/scrypt` to `projects/password-hashes`.

Reply:
I think we'd prefer to have this an explicit connection to the scrypt crate and keep the scrypt project name. Would you prefer to make the harness selection more constrained in this case?

Reply:
I actually make this dynamic to allow adding more fuzzers for the the scrypt crate without the need to modify the OSS-Fuzz integration later. But I do agree that using specific cp command is also OK. I am OK with either way.

Reply:
In that case it's worth to copy only binaries which satisfy the `scrypt*` regexp.

Reply:
Thanks for the suggestion. I have fixed that.